### PR TITLE
Improve some FIXME

### PR DESCRIPTION
**Pull request by Kvicii**
motivation:
Improve some FIXME codes

Revise:
Extract static variables instead of magic numbers, modify some documents

### CODE REVIEW

File: codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java [hunk 1]
```diff
@@ -36,19 +36,7 @@
...
-import static io.netty.handler.codec.compression.Lz4Constants.COMPRESSION_LEVEL_BASE;
-import static io.netty.handler.codec.compression.Lz4Constants.DECOMPRESSED_LENGTH_OFFSET;
-import static io.netty.handler.codec.compression.Lz4Constants.DEFAULT_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.DEFAULT_SEED;
-import static io.netty.handler.codec.compression.Lz4Constants.HEADER_LENGTH;
-import static io.netty.handler.codec.compression.Lz4Constants.MAGIC_NUMBER;
-import static io.netty.handler.codec.compression.Lz4Constants.MAX_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.MIN_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.TOKEN_OFFSET;
+import static io.netty.handler.codec.compression.Lz4Constants.*;
```
Comment (hyperxpro) [inline codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java:39]: The wildcard import is not allowed. Revert.

**Review by hyperxpro — CHANGES_REQUESTED** [review]

File: codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java (hunk 1 above)
Comment (Kvicii) [reply codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java:39]: I used netty code style, but it changed after importing with shortcut key.

Comment (Kvicii) [reply codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java]: @hyperxpro fixed.

**Comment by normanmaurer** [comment]
@Kvicii did you sign our ICLA yet? https://netty.io/s/icla

**Comment by Kvicii** [comment]
@normanmaurer 
Some time ago, I remember that I followed the documentation and signed the agreement, now do I need to sign it again?

**Comment by Kvicii** [comment]
@normanmaurer 
I signed again, please confirm, thx😀

**Comment by normanmaurer** [comment]
Nope all good

**Comment by normanmaurer** [comment]
@Kvicii thx!

**Comment by Kvicii** [comment]
@normanmaurer 
ok, thx.

**Pull request merged**